rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -746,15 +740,14 @@ void game_list_frame::Refresh(const bool from_drive, const bool scroll_after)
 					// The patch is game data and must have the same serial and an app version
 					if (entry->info.serial == other->info.serial && other->info.category == "GD" && other->info.app_ver != cat_unknown_localized)
 					{
-						try
 						{
 							// Update the app version if it's higher than the disc's version (old games may not have an app version)
-							if (entry->info.app_ver == cat_unknown_localized || std::stod(other->info.app_ver) > std::stod(entry->info.app_ver))
+							if (entry->info.app_ver == cat_unknown_localized || other->info.app_ver > entry->info.app_ver)
 							{
 								entry->info.app_ver = other->info.app_ver;
 							}
 							// Update the firmware version if possible and if it's higher than the disc's version
-							if (other->info.fw != cat_unknown_localized && std::stod(other->info.fw) > std::stod(entry->info.fw))
+							if (other->info.fw != cat_unknown_localized && other->info.fw >entry->info.fw)
```

Comment:
```suggestion
							if (other->info.fw != cat_unknown_localized && other->info.fw > entry->info.fw)
```

Reply:
I don't think this works. String compare is not equal number compare

Reply:
I think it does actually.

Reply:
I'm pretty sure that was my first implementation and it didn't work

Reply:
For example: "1.10" != "01.10"

Reply:
Rewrote it to actually convert to doubles